techniques/general/G173.html

```diff
@@ -1,10 +1,12 @@
-      <p>Any technology that supports audio and video.</p>
+      <p>Any technology that supports synchronized audio and video.</p>
    </section><section id="description"><h2>Description</h2>
-      <p>The objective of this technique is to provide a second version of video content that provides audio desciptions so that it is possible for people who cannot see to be able to understand audio-visual material.</p>
-      <p>Since most user agents today cannot merge multiple sound tracks, this technique adds the additional audio information to synchronized media by providing a second version of the movie where the original soundtrack and additional audio description have been combined in a single track. This additional information focuses on actions, characters, scene changes and on-screen text (not captions) that are important to understanding the content.</p>
-      <p>Since it is not helpful to have this new information obscure key audio information in the original sound track (or be obscured by loud sound effects), the new information is added during pauses in dialogue and sound effects. This limits the amount of supplementary information that can be added to program.</p>
-      <p>Providing a second version of the movie that includes audio descriptions as the primary sound track will make this content accessible for blind people who need to hear not only the dialogue, but also the context and other aspects of the video that are not communicated by the characters' dialogue alone.</p>
+      <p>The objective of this technique is to provide audio description in video content, so that it is possible for people who cannot see to understand audio-visual material.</p>
+      <p>Since most user agents today cannot merge multiple soundtracks, this technique adds audio description information into the single audio track of a movie. The added information focuses on actions, characters, scene changes, and on-screen text (not captions) that are important to understanding the content.</p>
+      <p>Since it is not helpful to have this new information obscure key audio information in the original soundtrack (or be obscured by loud sound effects), the new information is added during pauses in dialogue and sound effects. This limits the amount of supplementary information that can be added to the program.</p>
+      <p>Providing audio description in the movie's soundtrack will make this content accessible for blind people who need to hear not only the dialogue, but also the context and other aspects of the video that are not communicated by the characters' dialogue alone.</p>
```

Comment:
This presupposes that the characters' dialogue is the only possible source of other context within the original soundtrack.

```suggestion
      <p>Providing audio description in the movie's soundtrack will make this content accessible for blind people who need to hear not only the dialogue, but also the context and other aspects of the video that are not communicated in the original soundtrack.</p>
```